### ISSUE TITLE
Update for Classic WoW?

### ISSUE THREAD

**Issue by St4tikk** [Orukxu/SpamThrottle#18]
Hi Mopar!  Your addon was always a must have for me.  You implemented a reverse filter feature for me back on Nostalrius where all chat was filtered unless it contained a key word in the message.  Worked amazing!  Anyways I was curious if you have any plans to adapt this addon to work in Classic WoW or if you will be playing?

**Comment by Orukxu** [comment]
Yes absolutely. On retail, I'll be rolling it forward to the current API version, which is similar to my retail version (that I stopped maintaining at about MoP since Blizzard had put most capabilities into the game itself). It will have all the same capabilities as the current version (specifically, very good anti-gold-ad features), but I'll re-implement the chat filter approach since the retail API had a more sophisiticated (and much more stable) way of implementing chat handlers.
That will make it fully compatible with addons like Prat and Wim. Again. lol.

**Comment by St4tikk** [comment]
That is awesome news!!  Thanks so much for maintaining it.  Do you know what server you're going to be on yet?  I'm on Fairbanks at the moment but may end up playing on one of the Oceanic servers if the lag isn't to bad to meet up with guildies from vanilla.

**Comment by Orukxu** [comment]
Most likely on Herod, as that is where my guild is going. Looks like Blizzard is saying it's hugely overpopulated (which means there will be large queues); if it falls apart I'd probably go to the one they just opened given the unexpectedly high demand (can't remember the name).

**Comment by henriklb** [comment]
Any news on the Classic implementation?

**Comment by gotmilk0112** [comment]
Yeah, any update? SpamThrottle was the absolute best spam addon, and the temp-ignore thing was really good as well.

**Comment by shershenator** [comment]
I'm also waiting for update, please please

**Comment by TerjeKir** [comment]
Loved this addon on nost, hope to see an update :)
